### DISCUSSION TITLE
[Step 2: Obtain an authorization code] is not working now ?

### DISCUSSION THREAD

**Discussion by tenamynnek** [timdorr/tesla-api#419] [General]
it always work before..
but i found it's return "200" instead of "302" now ?

**Comment by jklaverstijn** [comment]
I can confirm this. My code seems broken now. Found out 1st of August. Perhaps related to the Captcha issue?

**Reply by tamemin** [reply]
Yes, I've just discovered this too, and I've only just implemented my code. I seem to be unlucky.

So, how do we stop ourselves looking like a robot, when we actually are a robot?

**Comment by LogixDE** [comment]
Any solutions?
